datagrepper/templates/base.html

```diff
@@ -83,7 +83,14 @@
     <script src="{{ url_for('static', filename='jquery-2.1.0.min.js') }}"></script>
     {% if autoscroll %}
     <script src="{{ url_for('static', filename='autoscroll.js') }}"></script>
+    {% if websocket_address is defined %}
```

Comment:
Do we really want the ``is defined`` part? If the variable is not set, or is null, jinja will treat it in the same way.